### PR TITLE
donot trigger ci when only docs changed

### PR DESCRIPTION
**Pull request by gy95**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
donot trigger ci when only docs changed
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: .github/workflows/cifuzz.yml [hunk 1]
```diff
@@ -1,5 +1,14 @@
 name: CIFuzz
-on: [pull_request]
+on: 
+  push:
+    branches:
+      - master
+    tags:
+    paths-ignore:
+      - '**.md'
```
Comment (vincentgoat) [inline .github/workflows/cifuzz.yml:8]: Adding this one?
- 'docs/**'
- 'hack/**'
- 'CHANGELOG/**'

Comment (gy95) [reply .github/workflows/cifuzz.yml:8]: files need ignoring include the below files.
docs/**(include some images) **.md OWNERS MAINTAINERS
I think this is all.

File: .github/workflows/cifuzz.yml [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+      - 'docs/**'
+      - 'OWNERS'
+      - 'MAINTAINERS'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
```
Comment (fisherxu) [inline .github/workflows/cifuzz.yml:15]: Small question, `**` is the grammer in github action? :) @gy95

File: .github/workflows/cifuzz.yml (hunk 1 above)
Comment (fisherxu) [reply .github/workflows/cifuzz.yml:8]: > 'hack/**'

There are many scripts under hack dir, they need to run the ci check. :)

Comment (fisherxu) [reply .github/workflows/cifuzz.yml:8]: There are many scripts under hack, they need to be checked. :)

Comment (gy95) [reply .github/workflows/cifuzz.yml:8]: yes, hack dir should be checked

File: .github/workflows/cifuzz.yml (hunk 2 above)
Comment (gy95) [reply .github/workflows/cifuzz.yml:15]: can match more string than only one `*`
ref: https://docs.github.com/cn/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Comment (gy95) [reply .github/workflows/cifuzz.yml:15]: /hold need some fix

Comment (fisherxu) [reply .github/workflows/cifuzz.yml:15]: Is `**/OWNERS` needed?

Comment (gy95) [reply .github/workflows/cifuzz.yml:15]: /hold cancel
fixed

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**